### PR TITLE
Interactive Done

### PR DESCRIPTION
**Pull request by oohwee-yju**
- [v] Read [Contribute to NTHU-CPP](https://nthu-cp.github.io/NTHU-CPP/others/contribution.html).
- [v] Rebase to the latest main branch.
- [v] List all the references.
- [v] Successfully build the website by mdBook with no error after the change.
- [v] Exclude any irrelevant files.
- [v] Link to the issue which is related to your change.
- [v] Review the content by myself.
- [v] Pass tests/CI.

### CODE REVIEW

**Comment by oohwee-yju** [comment]
no, this is final version

File: src/misc/interactive.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+### Example: Binary Search
+
+:::
+Jury 藏著一個數字 \\(x\\)，Participant 可以問 Jury 一個數字 \\(y\\)，Jury 會回答以下三種情況是否滿足：
+
+- \\(x < y\\)
+- \\(x > y\\)
+- \\(x = y\\)
+
```
Comment (harry900831) [inline src/misc/interactive.md:27]: It seems that the contend isn't correctly be rendered on mdBook. 
Please revise it.

File: src/misc/interactive.md [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+#### 把 query 包成一個 function
+
+其實大部份時候會問的 query 都會符合某些 pattern，把它包成 function 可以有效降低 coding 的複雜度。
+
+就算 query 沒有什麼規律可言，包成 function 也可以減少在主要的 procedure 中處理互動，會讓 code 好看一點。
+
+### Binary Search
+
+> [Codeforces 809B - Glad to see you!](https://codeforces.com/problemset/problem/809/B)
+> Jury 有一組大小為 \\(k\\) 的集合，集合內的元素為 \\(1 \sim n\\) 且不重複的數字。> Jury 會將 \\(k\\) 跟 \\(n\\) 告訴 Participant。> Participant 可以問 Jury 的問題是：給兩個數字 \\(x, y\\)，Jury 將會在它的集合中找到跟 \\(x\\) 最接近的數字 \\(a\\)、跟 \\(y\\) 最接近的數字 \\(b\\)（\\(a\\) 可能等於 \\(b\\)），並回答是否滿足 \\(|x-a| \le |y-b|\\)。> Participant 需要在 \\(60\\) 個詢問內找到該集合中的任意兩個數字，其中 \\(2 \le k \le n \le 10^5\\)。
```
Comment (harry900831) [inline src/misc/interactive.md:90]: You should insert a blank line between the tile and the content

File: src/misc/interactive.md
Comment (harry900831) [inline src/misc/interactive.md:1]: Would you like to share the ppt that you teach before?
As a reference or put it at the beginning

**Review by harry900831 — CHANGES_REQUESTED** [review]
- It seems that the content isn't correctly rendered on mdBook
- I may merge this after you revise it, please scan through the whole article and check whether there's any typo or error.

File: src/misc/interactive.md (hunk 1 above)
Comment (harry900831) [reply src/misc/interactive.md:27]: Render error not only on this part, please check the whole article.

**Comment by oohwee-yju** [comment]
Problems should be fixed.
But I would like to keep the ppt private.

**Comment by harry900831** [comment]
I'll merge it after you fix the last issue

**Comment by oohwee-yju** [comment]
I didn't found any typo.

**Review by harry900831 — CHANGES_REQUESTED** [review]

**Comment by harry900831** [comment]
It's seems that my review last week isn't out there so you didn't see it. I'll merge it and fix it by myself. Thanks

**Pull request merged**